hash/msx2_flop.xml

```diff
@@ -12827,7 +15529,10 @@ The following floppies came with the machines.
 	<software name="ysa" cloneof="ys">
 		<description>Ys - Ancient Ys Vanished Omen (Japan, alt)</description>
 		<year>1987</year>
-		<publisher>Falcom</publisher>
+		<publisher>Sony</publisher>
+		<info name="alt_title" value="イース"/>
```

Comment:
Is the subtitle “Ancient Ys Vanished Omen” actually used anywhere?  Different releases used “Ancient Ys Vanished” and “The Vanished Omens”.  As far as I know, the MSX2 box art says “Ancient Ys Vanished” (the original PC-8801 box art was used).

Reply:
Yes, on the title screen: 
![](https://images.generation-msx.nl/software_title/68304492.png)